test/browser.js

```diff
@@ -171,27 +172,48 @@ test('throws if does not support ReadableStream', withPage, async (t, page) => {
 	await server.close();
 });
 
-test('FormData with searchParams', withPage, async (t, page) => {
-	t.plan(2);
-
+test.failing('FormData with searchParams', withPage, async (t, page) => {
+	t.plan(6);
 	const server = await createTestServer();
 	server.get('/', (request, response) => {
 		response.end();
 	});
 	server.post('/', async (request, response) => {
-		const requestBody = await pBody(request);
-
-		t.regex(requestBody, /bubblegum pie/);
 		t.deepEqual(request.query, {foo: '1'});
+		const body = await new Promise((resolve, reject) => {
+			const busboy = new Busboy({headers: request.headers});
+
+			busboy.on('file', (fieldname, file, filename, encoding, mimetype) => {
+				let fileContent = '';
+
+				file.on('data', data => {
+					fileContent += data.toString('utf-8');
+				});
 
+				file.on('end', () => {
+					resolve({fieldname, filename, encoding, mimetype, fileContent});
+				});
+
+				file.on('error', reject);
+			});
+
+			busboy.on('error', reject);
+
+			request.pipe(busboy);
+		});
+		t.is(body.fieldname, 'file');
+		t.is(body.filename, 'my-file');
+		t.is(body.encoding, '7bit');
+		t.is(body.mimetype, 'text/plain');
+		t.is(body.fileContent, 'bubblegum pie');
```

Comment:
I think we should use `t.deepEqual()` to ensure we are asserting _all_ properties of `body`.